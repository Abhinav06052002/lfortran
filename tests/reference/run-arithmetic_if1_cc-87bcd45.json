{
    "basename": "run-arithmetic_if1_cc-87bcd45",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/arithmetic_if1_cc.f90",
    "infile_hash": "d43842d19860f048f4fed4bcf95f6f2a2fa8efa8b3292d8da86ce349",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-arithmetic_if1_cc-87bcd45.stdout",
    "stdout_hash": "605554f0634876818c22a6dd5ecf4c230919c0ec9b1e18af30597d80",
    "stderr": "run-arithmetic_if1_cc-87bcd45.stderr",
    "stderr_hash": "5c4a85236775b3848107d1312a3ee059c7f27f38bfc23df194a26648",
    "returncode": 2
}