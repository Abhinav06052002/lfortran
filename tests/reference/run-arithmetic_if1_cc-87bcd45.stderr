semantic error: Arithmetic if (x) requires an integer or real for `x`
 --> tests/errors/arithmetic_if1_cc.f90:5:9
  |
5 |     if ("yy") 1, 2, 3
  |         ^^^^ 
